chaoslib/litmus/platform/gke/node_memory_consumption.yml

```diff
@@ -0,0 +1,148 @@
+ ## GETTING THE PRE CHAOS NODE NAME OF APPLICATION 
+- block:
+  
+    - name: Getting the pre-chaos node name of application 
```

Comment:
Getting the pod name of the application (pre chaos), here we are only getting pod name